### PR TITLE
Connecting-socket-tot-the-board

### PR DESCRIPTION
**Pull request by OriGiladi**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    gameArray?: cardData[][];
+    firstTeamWords?: string[];
+    secondTeamWords?: string[];
+    civilianWords?: string[];
+    assassinWord?: string [];
+    turn?: team;
+    firstTeam?: team;
+    secondTeam?: team;
+    codeMasterView?: boolean;
+    guessPhase?: boolean;
```
Comment (arielcoral) [inline src/index.ts]: Do we need this?

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    });
+
+    socket.on('disconnect', () => {
+        console.log('🔥: A user disconnected');
+        users.forEach(user => {
+            console.log(user.socketID, socket.id)
+        })
+        
+        users = users.filter(user => user.socketID !== socket.id);
+        socketIO.emit('newUserResponse', users);
```
Comment (arielcoral) [inline src/index.ts]: "newUserResponse" i dont like the name because we are not adding a new user

Comment (OriGiladi) [reply src/index.ts]: i changed it to 'updatingUsersResponse', hope it's ok

File: src/index.ts (hunk 1 above)
Comment (OriGiladi) [reply src/index.ts]: you are right, removing

**Pull request closed without merging**